Training_Center_Task_11/Pages/HomePage.cs

```diff
@@ -0,0 +1,44 @@
+﻿using OpenQA.Selenium;
+
+namespace Training_Center_Task_11.Pages
+{
+    public class HomePage : MainPage
+    {
+        private IWebDriver _driver;
```

Comment:
is it needed if  driver located in base page

Reply:
Not really :D Fixed.